nodejs/week2/mealSharing/src/bakckend/routes/reservations.js

```diff
@@ -0,0 +1,25 @@
+const express = require("express");
+const router = express.Router();
+const reservations = require("../data/reservations.json");
+
+router.get("/:id", (req, res) => {
+	const reserveId = parseInt(req.params.id);
+	if (!reserveId) {
+		res.send(`Enter an id number`);
+	} else {
+		const reservationWithThisId = reservations.find((r) => r.id === reserveId);
+		if (!reservationWithThisId) {
+			res.send(`There is no reservation with this id: ${reserveId}`);
```

Comment:
In REST there's a formal way in which to tell a client that a resource with a given id doesn't exist.

Try to see how to do that with express: Send a 404 resource not found response